src/main/resources/application.yaml

```diff
@@ -75,16 +75,24 @@ service-config:
     - service: probate
       jurisdiction: PROBATE
       transformation-url: ${TRANSFORMATION_URL_PROBATE}
-      case-type-ids: []
+      case-type-ids:
+        - GrantOfRepresentation
+        - Caveat
+        - StandingSearch
+        - WillLodgement
       allow-creating-case-before-payments-are-processed: true
 
     - service: divorce
       jurisdiction: DIVORCE
-      case-type-ids: []
+      case-type-ids:
+        - DIVORCE
+        - DIVORCE_NOTICE_OF_ACTING
```

Comment:
This one is to be confirmed by Divorce (they're going to come back to us), but for now it's better to stay on the safe side and include it.